compiler/cpp/test/lexer/LexerTest.cpp

```diff
@@ -0,0 +1,435 @@
+//
+// Created by Marc on 30.05.2021.
+//
+
+#include <vector>
+#include <lexer/Lexer.h>
+#include <gtest/gtest.h>
+#include "../util/LanguageDescriptor.h"
+#include "../util/FileReader.h"
+
+struct LexerParams {
+    const std::string fileName;
+    std::vector<TokenType> expectedTokensLineComment;
+    std::vector<TokenType> expectedTokensBlockComment;
+};
+
+const std::vector<LanguageDescriptor> LANGUAGES {
+    LanguageDescriptor("yml", "#", "", ""),
+    LanguageDescriptor("cpp", "//", "/*", "*/"),
+    LanguageDescriptor("html", "", "<!--", "-->"),
+    LanguageDescriptor("py", "#", "", "")
```

Comment:
Python has block identifiers `'''`

Reply:
Oh snap! Forgot about that ... Good catch

Reply:
Fixed it!